src/lando/api/legacy/workers/landing_worker.py

```diff
@@ -230,6 +244,20 @@ def apply_patch(revision: Revision):
 
         self.update_repo(repo, job, scm, job.target_commit_hash)
 
+        if job.is_pull_request_job and job.handover_repo and job.handover_repo.is_try:
```

Comment:
I'd suggest moving the `is_try` down, as a failing case, so we don't continue over this check if something is inconsistent
```suggestion
        if job.is_pull_request_job and job.handover_repo:
            if not job.handover_repo.is_try:
                raise Exception("Cannot hand-over to non-try repos.")
```